streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueStoreSupplier.java

```diff
@@ -58,28 +59,24 @@ public String name() {
     }
 
     public StateStore get() {
-        return new MeteredKeyValueStore<>(new MemoryStore<K, V>(name).enableLogging(serdes), "in-memory-state", time);
+        return new MeteredKeyValueStore<>(new MemoryStore<K, V>(name, keySerde, valueSerde).enableLogging(), "in-memory-state", time);
```

Comment:
If this is registering a metric that we expose, do we need to update docs to include this new metric?  If yes, then regardless of when/where we update the docs (i.e. in this PR or in a different JIRA/PR) we should not forget about it.